tests/test_symbiyosys/config.sby.j2

```diff
@@ -0,0 +1,21 @@
+[tasks]
+prove pf
+cover cv
+
+[options]
+pf: mode prove
+pf: depth 3
+
+cv: mode cover
+cv: depth 32
+
+[engines]
+smtbmc boolector
+
+[script]
+{{"-sv"|gen_reads}}
+
+prep -top my_top_module
```

Comment:
Can you use a toplevel template variable here?

Reply:
Done! And, amusingly, I'd got the name wrong - it should have just been `top_module`.

Reply:
Yeah, I saw that :-)